js/app.js

```diff
@@ -0,0 +1,156 @@
+//this has to be outside document ready so the click events listeners 
+//can access after docready executes
+var quiz = []; 
+var totalScore = 0;
+var whichQ = 0;
+
+$(document).ready(function(){
+
+
+//init quiz
+loadQuestions();
+drawQuiz();
+
+
+
+//display first question
+$(".intro-start").click(function(){
+	//move intro offscreen
+	nextSlide(this);
+});
+
+//next question
+$(".answer").click(function(){
+	//score if not out of questions 
+	if(whichQ < quiz.length){
+		quiz[whichQ].storeAndCompare(this.id);
+	}
+	//if last question, update totalScore scorecard
+	if(whichQ===quiz.length-1) {
+		$("#scorecard>h3").text(totalScore+"/"+quiz.length);
+	}
+	//move question offscreen
+	$(".carousel-item").each(function(){
+		if($(this).offset().left <0) {
+			$(this).css("left","150%");
+		}
+	});
+	//move to nextslide
+	nextSlide(this);
+
+
+	//increment question counter
+	whichQ++;
+});
+
+//restart game
+$(".scorecard-restart").click(function(){
+
+	totalScore=0;
+	whichQ=0;
+	quiz.forEach(function(quest){
```

Comment:
the reset functionality for each question could be another function on the Question object as indicated below.